service/src/main/kotlin/fi/espoo/evaka/serviceneednew/NewServiceNeedQueries.kt

```diff
@@ -61,6 +61,16 @@ val migrationSqlCases = """
         WHEN p.type = 'DAYCARE' AND sn.hours_per_week <= 25 AND NOT sn.part_day AND sn.part_week THEN 'Osaviikkoinen, enintään 25h'
         WHEN p.type = 'DAYCARE_PART_TIME' AND sn.hours_per_week <= 25 AND sn.part_day AND NOT sn.part_week THEN 'Osapäiväinen'
         WHEN p.type = 'DAYCARE_PART_TIME' AND sn.hours_per_week <= 25 AND sn.part_day AND sn.part_week THEN 'Osapäiväinen ja osaviikkoinen'
+        WHEN p.type = 'DAYCARE_FIVE_YEAR_OLDS' AND sn.hours_per_week > 35 AND sn.hours_per_week < 45 AND NOT sn.part_day AND NOT sn.part_week THEN '5-vuotiaiden kokopäiväinen, vähintään 45h'
+        WHEN p.type = 'DAYCARE_FIVE_YEAR_OLDS' AND sn.hours_per_week > 35 AND sn.hours_per_week < 45 AND NOT sn.part_day AND NOT sn.part_week THEN '5-vuotiaiden kokopäiväinen, 35-45h'
```

Comment:
Aren't these two conditions identical? Based on the output, I'd assume this should be:

```suggestion
        WHEN p.type = 'DAYCARE_FIVE_YEAR_OLDS' AND sn.hours_per_week >= 45 AND NOT sn.part_day AND NOT sn.part_week THEN '5-vuotiaiden kokopäiväinen, vähintään 45h'
        WHEN p.type = 'DAYCARE_FIVE_YEAR_OLDS' AND sn.hours_per_week > 35 AND sn.hours_per_week < 45 AND NOT sn.part_day AND NOT sn.part_week THEN '5-vuotiaiden kokopäiväinen, 35-45h'
```

And more out of curiosity: do we generally prefer `a > x AND a < y` vs. `a BETWEEN x+1 AND y-1`? IMO `BETWEEN` is more readable _but_ the requirement to always use "*-or-equal" logic can be confusing (e.g. here you would have 45 in output and 44 in condition).

Reply:
Thanks for catching the incorrect condition, this stuff is pretty hard to read 😄 

`a BETWEEN x+1 AND y-1` can't be used here anyway since the `sn.hours_per_week` values are not integers. I agree that `BETWEEN` is more readable in general.